ui/hooks/src/utils/entry-utils.ts

```diff
@@ -27,15 +34,15 @@ function fromBinary(binary: any) {
   return result;
 }
 
-export const serializeBase64ToSlate = (base64Content: string, logger?: any) => {
-  const stringContent = atob(base64Content);
+export const decodeb64SlateContent = (base64Content: string, logger?: ILogger) => {
+  const stringContent = window.atob(base64Content);
```

Comment:
Why did you use it as a method (window.atob) instead of directly?

Reply:
to avoid deprecation warning, atob is deprecated in nodejs. i can avoid that warning explicitly using window.atob